### PR TITLE
AC Automated Test - LRAC-9095 | master

### PR DESCRIPTION
**Pull request by alanafergs**
https://issues.liferay.com/browse/LRAC-9095

### CODE REVIEW

**Comment by alanafergs** [comment]
ci:test:sf

**Comment by alanafergs** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/UserManagement.testcase [hunk 1]
```diff
@@ -77,12 +77,47 @@ definition {
...
+				ACSettings.changeMultipleUsersPermissions(
+					permissionType = "Administrator",
+					userEmail = "ac@liferay.com");
+			}
+
+			task ("Assert user permission") {
+				ACSettings.assertUserPermission(
+					permissionType = "Administrator",
+					userEmail = "ac@liferay.com");
+			}
```
Comment (ViniLopes20) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/UserManagement.testcase]: Hi @alanafergs, When you are going to do another PR, you could put in your test the macro to delete users that were created by Mateus [here](https://github.com/liferay-qa/liferay-portal/pull/943/commits/c750cbb297b35948ab45228e4d75f15863a5c32a)

Comment (alanafergs) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/UserManagement.testcase]: Ok!

**Comment by alexcandidos** [comment]
hi @ViniLopes20 , thanks for helping to review it.

Hi @alanafergs  , Thanks for working on it!

The code looks good to me!

I'm forwarding it.

**Comment by alexcandidos** [comment]
ci:forward
